.github/workflows/cmake-all.yml

```diff
@@ -1,6 +1,6 @@
 name: CMake all CGAL
 
-on: [push, pull_request]
+on: push
```

Comment:
Why have you removed `pull_request`?

Reply:
I think I changed the wrong file at some point, it's an error